### DISCUSSION TITLE
Minimum docker version requirement for 1.52.0

### DISCUSSION THREAD

**Discussion by lt2865** [jaegertracing/jaeger#5005] [Q&A]
I recently tried pulling the latest version of jaeger-collector and it was failing due to "missing signature key". This happens when there is a client side issue, which in my case was that my version of docker was too old. Once I upgraded to the latest version of docker (24.0.7), I was able to pull the image no problem.

Just wondering if anyone knows what the minimum version of docker that is required to pull and run the latest version of Jaeger?

Thanks

**Comment by akagami-harsh** [comment]
>Just wondering if anyone knows what the minimum version of docker that is required to pull and run the latest version of Jaeger?

i think its 19.03 for jaeger 1.52.0

**Comment by yurishkuro** [comment]
@lt2865 do you think it was specific to Jaeger or just between your Docker install and the registry? I.e. were you able to pull other images?

We don't really control which version of Docker we use during the build.

**Reply by lt2865** [reply]
It seemed to be as I was able to pull other images fine just not the jaeger ones for that version for some reason but yh once I upgraded to the latest version of docker I was able to pull the image fine
